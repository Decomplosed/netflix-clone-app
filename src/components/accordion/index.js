import React from 'react';

export default function Accordion({ children, ...restProps }) {
  return <Container></Container>;
}
