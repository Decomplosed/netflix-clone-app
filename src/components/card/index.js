import React from 'react';

export const FeatureContext = createContext();
