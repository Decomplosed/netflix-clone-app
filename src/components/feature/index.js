import React from 'react';
import { Container, Title, SubTitle } from './styles/feature';

export default function Feature({ children, ...restProps }) {
  return <Container></Container>
}
