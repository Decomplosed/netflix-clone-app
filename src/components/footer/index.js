import React from 'react';

export default function Footer({ children, ...restProps }) {
  return <Container {...restProps}></Container>;
}
