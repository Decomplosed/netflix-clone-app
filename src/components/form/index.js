import React from 'react';
import {
  Container,
  Base,
  Error,
  Title,
  Text,
  TextSmall,
  Link,
  Input,
  Submit,
} from './styles/form';

export default function Form({ children, ...restProps }) {
  return <Container {...restProps}>{children}</Container>;
}

Form.Base = function FormBase({ children, ...restProps }) {
  return <Base {...restProps}>{children}</Base>;
};

Form.Error = function FormError({ children, ...restProps }) {
  return <Error {...restProps}>{children}</Error>;
};

Form.Title = function FormTitle({ children, ...restProps }) {
  return <Title {...restProps}>{children}</Title>;
};
