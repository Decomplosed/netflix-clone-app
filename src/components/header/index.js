import React from 'react';

export default Header({bg = true, children, ...restProps}) {}