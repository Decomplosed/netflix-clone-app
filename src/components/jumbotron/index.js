import React from 'react';

export default Jumbotron = () => {
  return <p>I am jumbotron</p>;
};
