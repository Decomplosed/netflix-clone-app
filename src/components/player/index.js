import React, { useState, useContext, createContext } from 'react';
import ReactDOM from 'react-dom';
