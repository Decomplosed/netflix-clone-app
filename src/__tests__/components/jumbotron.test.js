import React from 'react';
import { render } from '@testing-library/react';
import { Jumbotron } from '../../components';
import jumboData from '../../fixtures/jumbo';

describe('<Jumbotron />', () => {
  it('Renders the <Jumbotron /> with populated data', () => {
    const { container, getByText, getByAltText, getByTestId } = render(
      <Jumbotron.Container>
        {jumboData.map((item) => (
          <Jumbotron key={item.id}>
            <Jumbotron.Pane>
              <Jumbotron.Title>{item.title}</Jumbotron.Title>
              <Jumbotron.SubTitle>{item.subTitle}</Jumbotron.SubTitle>
            </Jumbotron.Pane>
          </Jumbotron>
        ))}
      </Jumbotron.Container>,
    );
  });
});
