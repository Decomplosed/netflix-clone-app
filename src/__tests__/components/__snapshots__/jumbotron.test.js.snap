// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Jumbotron /> Renders the <Jumbotron /> with populated data 1`] = `
<div
  class="jumbotron__Container-fl0gp-6 huWUsu"
>
  <div
    class="jumbotron__Item-fl0gp-5 gapEpD"
  >
    <div
      class="jumbotron__Inner-fl0gp-0 BAPUh"
      direction="row"
    >
      <div
        class="jumbotron__Pane-fl0gp-1 dfYQVT"
      >
        <h1
          class="jumbotron__Title-fl0gp-2 fozQUO"
        >
          Enjoy on your TV.
        </h1>
        <h2
          class="jumbotron__SubTitle-fl0gp-3 gSNczd"
        >
          Watch on smart TVs, PlayStation, Xbox, Chromecast, Apple TV, Blu-ray players and more.
        </h2>
      </div>
      <div
        class="jumbotron__Pane-fl0gp-1 dfYQVT"
      >
        <img
          alt="Tiger King on Netflix"
          class="jumbotron__Image-fl0gp-4 gHOOHR"
          data-testid="1-jumbo-image"
          src="/images/misc/home-tv.jpg"
        />
      </div>
    </div>
  </div>
  <div
    class="jumbotron__Item-fl0gp-5 gapEpD"
  >
    <div
      class="jumbotron__Inner-fl0gp-0 BAPUh"
      direction="row"
    >
      <div
        class="jumbotron__Pane-fl0gp-1 dfYQVT"
      >
        <h1
          class="jumbotron__Title-fl0gp-2 fozQUO"
        >
          Download your programmes to watch on the go.
        </h1>
        <h2
          class="jumbotron__SubTitle-fl0gp-3 gSNczd"
        >
          Save your data and watch all your favourites offline.
        </h2>
      </div>
      <div
        class="jumbotron__Pane-fl0gp-1 dfYQVT"
      >
        <img
          alt="Watch on mobile"
          class="jumbotron__Image-fl0gp-4 gHOOHR"
          data-testid="2-jumbo-image"
          src="/images/misc/home-mobile.jpg"
        />
      </div>
    </div>
  </div>
  <div
    class="jumbotron__Item-fl0gp-5 gapEpD"
  >
    <div
      class="jumbotron__Inner-fl0gp-0 BAPUh"
      direction="row"
    >
      <div
        class="jumbotron__Pane-fl0gp-1 dfYQVT"
      >
        <h1
          class="jumbotron__Title-fl0gp-2 fozQUO"
        >
          Watch everywhere.
        </h1>
        <h2
          class="jumbotron__SubTitle-fl0gp-3 gSNczd"
        >
          Stream unlimited films and TV programmes on your phone, tablet, laptop and TV without paying more.
        </h2>
      </div>
      <div
        class="jumbotron__Pane-fl0gp-1 dfYQVT"
      >
        <img
          alt="Money Heist on Netflix"
          class="jumbotron__Image-fl0gp-4 gHOOHR"
          data-testid="3-jumbo-image"
          src="/images/misc/home-imac.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
